src/libraries/System.Collections.Immutable/src/System/Collections/Frozen/EmptyFrozenDictionary.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Runtime.CompilerServices;
+
+namespace System.Collections.Frozen
+{
+    /// <summary>Provides an empty <see cref="FrozenDictionary{TKey, TValue}"/> to use when there are zero key/value pairs to be stored.</summary>
+    internal sealed class EmptyFrozenDictionary<TKey, TValue> : FrozenDictionary<TKey, TValue>
+        where TKey : notnull
+    {
+        public EmptyFrozenDictionary() : base(EqualityComparer<TKey>.Default) { }
```

Comment:
Why is this one public, but the ctor for EmptyFrozenSet is internal?

Reply:
Left hand not remembering what the right hand did.  I'll change it.